srcpkgs/gnupg/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'gnupg'
```

Comment:
```suggestion
# Template file for 'gnupg'
# minor version updates (2.3-> 2.4) often need a fix in reverse dependencies
```
There has been a lot of packages that broke with the last few minor updates because their build systems hardcore supported major.minor versions of gnupg, I think we should include a comment like this to tell the person who is updating this package to be careful about it and fix dependants when necessary. They can be fixed with a trivial patch.